src/components/SlidePanel/SlidePanel.js

```diff
@@ -38,6 +38,7 @@ function SlidePanel({
     if (
       isActive &&
       !e.target.closest(".slide-panel") &&
+      !e.target.closest('[role="row"]') &&
```

Comment:
Drive-by to fix a bug where if you clicked on a row in the panel it would close the panel. We'll want a more robust system than this otherwise this list will continue to grow.